lms/static/sass/_build-lms-v1.scss

```diff
@@ -3,6 +3,12 @@
 
 // About: Sass compile for the LMS Elements that are shared between LTR and RTL UI. Configuration and vendor specific imports happen before this shared set of imports are compiled in the lms-main-*.scss files.
 
+// Bootstrap support for use with shared partials
+@import 'bootstrap/theme';
+@import 'bootstrap/scss/functions';
+@import 'bootstrap/scss/variables';
+@import "bootstrap/scss/mixins/breakpoints";
```

Comment:
Nice catch. I also did only a quick pass, plus I was spot checking on the sandbox. IMO the Bootstrap colors are the correct ones as they were taken from the Pattern Library. I should probably remove the duplicates so that this is clearer.